docs/source/wakepy-mode-lifecycle.md

```
@@ -1,7 +1,12 @@
 # Wakepy Mode Lifecycle
 
 ## Introduction to Modes
-[**Modes**](#wakepy-modes) are what you enter, stay in for a while, and exit from. For example, `keep.running` is a Mode where automatic suspend is inhibited. Modes can be used in two ways. As **context managers**:
+
+[**Modes**](#wakepy-modes) are what you enter, stay in for a while, and exit from. For example, [keep.running](#keep-running-mode) is a Mode where automatic suspend is inhibited. Each Mode is implemented with one or more [Methods](#wakepy-methods) like [org.gnome.SessionManager](#org-gnome-sessionmanager) or [SetThreadExecutionState](#windows-stes).
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)